hopsworks-common/src/main/java/io/hops/hopsworks/common/elastic/ElasticJWTController.java

```diff
@@ -73,16 +73,16 @@ public String createTokenForELKAsDataOwner(Project project)
   }
   
   public String createTokenForELKAsAdmin() throws ElasticException {
-    return createTokenForELK("admin", ElasticSettings.ELASTIC_ADMIN_ROLE);
+    return createTokenForELK("admin", null, ElasticSettings.ELASTIC_ADMIN_ROLE);
   }
   
   private String createTokenForELK(Project project, String role)
       throws ElasticException {
     String userRole = ElasticUtils.getValidRole(role);
-    return createTokenForELK(project.getName(), userRole);
+    return createTokenForELK(project.getName(), project.getInode().getId(), userRole);
   }
   
-  private String createTokenForELK(String project, String userRole)
+  private String createTokenForELK(String project, Long projectInodeId, String userRole)
```

Comment:
I think this would benefit of an Optional

Reply:
fixed